.github/workflows/doc-checks.yml

```diff
@@ -34,5 +36,8 @@ jobs:
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
 
+      - name: Build start
+        run: pnpm run build
+
```

Comment:
We should have one source of truth for the docs checks. It's easy for this file and `.github/workflows/deploy-docs.yml` to get out of sync.